lib/t/push_notifications/dispatch_job.ex

```diff
@@ -296,6 +296,15 @@ defmodule T.PushNotifications.DispatchJob do
     :ok
   end
 
+  defp handle_type(type, _args)
+       when type in ["newbie_live_mode_today", "newbie_live_mode_soon"] do
+    Feeds.newbies_list_today_participants()
```

Comment:
Oops, that's bad since when it runs at 13:00 (for `newbie_live_mode_today`) it would include yesterdays newbies. I'll need to fix it.

Reply:
fixed